packages/react-components/src/components/chart/baseChart.tsx

```diff
@@ -98,7 +98,7 @@ const BaseChart = ({
     thresholds,
     utilizedViewport,
     visibleData,
-  } = useVisualizedDataStreams(queries, viewport);
+  } = useVisualizedDataStreams(queries, viewport, undefined);
```

Comment:
is it important to pass undefined here? Are we using this hook in any other components? Maybe we can just remove that param entirely?

Reply:
Nope, that should be removed since im not passing styleSettings as a parameter there